packages/projection-typeorm/test/operators/storeHandles.test.ts

```diff
@@ -374,5 +374,44 @@ describe('storeHandles', () => {
         handle
       });
     });
+
+    it('handles error gracefully when minting an assetName in cip67 format', async () => {
+      const mintTxId = Cardano.TransactionId('0000000000000000000000000000000000000000000000000000000000000000');
+      const repository = queryRunner.manager.getRepository(HandleEntity);
+      const invalidAssetId = Cardano.AssetId(
+        'f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a000de140746573745f6d655f30303031'
+      );
+      const testAddress = Cardano.PaymentAddress(
+        'addr_test1qz690wvatwqgzt5u85hfzjxa8qqzthqwtp7xq8t3wh6ttc98hqtvlesvrpvln3srklcvhu2r9z22fdhaxvh2m2pg3nuq0n8gf2'
+      );
+      const newHandle = 'test_me_0001';
+
+      const source$ = createMultiTxProjectionSource([
+        {
+          body: {
+            fee: 111n,
+            inputs: [],
+            mint: new Map([[invalidAssetId, 1n]]),
+            outputs: [
+              {
+                address: testAddress,
+                value: {
+                  assets: new Map([[invalidAssetId, 1n]]),
+                  coins: 123n
+                }
+              }
+            ]
+          },
+          id: mintTxId
+        }
+      ]);
+      await firstValueFrom(source$.pipe(applyOperators()));
+      expect(
+        await repository.findOne({
+          select: { cardanoAddress: true, handle: true },
+          where: { handle: newHandle }
+        })
+      ).toEqual(null);
```

Comment:
I believe the test case from the task should actually be saved in the projection, but the asset name should be decoded first.
You can use the decode function from packages/core/src/Asset/cip67.ts to extract the asset name content and validate against that value.

Reply:
I raised this issue with @rhyslbw, but he didn't agree.

@rhyslbw thoughts?

Reply:
This is exactly the approach I suggested. :)

Reply:
The idea is not to add support for cip67 NFTs (yet) but solidify the existing implementation to be resilient to unexpected `assetNames`. I've [suggested a change](https://github.com/input-output-hk/cardano-js-sdk/pull/777#discussion_r1239699877) to make the test relevant once we add support for the new format though

Reply:
Ok, understood.